doc/api/fs.md

```diff
@@ -1646,7 +1776,7 @@ changes:
     description: The `cache` parameter was removed.
 -->
 
-* `path` {string|Buffer};
+* `path` {string|Buffer|URL};
```

Comment:
I think the `;` here is a typo, you can remove it while you’re already here if you want